taxonomy-editor-frontend/src/pages/startproject/index.jsx

```diff
@@ -0,0 +1,113 @@
+import { Typography, Box, Grid, TextField, Stack, Autocomplete, Snackbar, Alert, CircularProgress, Button } from "@mui/material";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { TAXONOMY_NAMES } from "../../constants";
+import { createBaseURL } from "../editentry/createURL";
+import { toSnakeCase } from "../editentry/interConvertNames";
+
+const StartProject = () => {
+    const [branchName, setBranchName] = useState("")
+    const [taxonomyName, setTaxonomyName] = useState(null)
+    const [description, setDescription] = useState("");
+    const [loading, setLoading] = useState(false); 
+    const [errorMessage, setErrorMessage] = useState(null);
+    const navigate = useNavigate();
+    
+    const handleSubmit = () => {
+        const baseUrl = createBaseURL(taxonomyName, branchName)
+        setLoading(true);
+        const dataToBeSent = {'description' : description};
+        fetch(baseUrl+'import', {
+            method : 'POST',
+            headers: {"Content-Type" : "application/json"},
+            body: JSON.stringify(dataToBeSent)
+        }).then(async (response) => {
+            const responseBody = await response.json();
+            if (!response.ok) {
+                throw Error(responseBody.detail)
```

Comment:
guess the synthax should be `throw new Error()`. From a personal perspective, I think is more error prone to use `throw` inside promises/async, I honestly prefer `Promise.reject(new Error(...))` as I know it will returned as  a promise rejection, instead of a thrown error that will be hanled as a promise rejection (but in some cases is not)

Reply:
are we 100% sure that all responseBody that are not ok will have a `.detail` ? (like for example what happens if is an error that is not happening in the server, or that we are not sending, etc...) maybe use another check for that `if (!response.ok && responseBody.detail) {`

Reply:
It would be always having a detail, but I'll anyways add the check

Reply:
I think there's actually no way to be 100% sure about it, like if we control the response body then it will have it if we add it, but it can be `!ok` for reasons outisde of our control (server errors, network connections, etc) and in those circumstances it won't be possible to know if that will happen, which is the case for all errors (you can be sure of what you do/throw, but it can fail for whatever reason other than the cases you control)